### ISSUE TITLE
IPM analysis years moved <2018 IPM won't run 

### ISSUE THREAD

**Issue by nort0160** [Huh/PopR_SDGFP#251]
I was trying to run IPM analysis years for WTD from 2009-2017 and it looks like when I move the slider before 2018 I lose license information and am unable to run IPM.

**Comment by Huh** [comment]
Yes, I need to clean up the handling of the harvest data when future
projections don't exist.




[image: speedgoat logo] <http://www.speedgoat.io/>
*Josh Nowak*, President
*Speedgoat Wildlife Solutions*
(406) 214-7666

   - [image: Check out my work on Github!] <https://github.com/Huh>
   - [image: Connect with me on LinkedIn!]
   <https://www.linkedin.com/in/josh-nowak-speedgoat/>

**Comment by Huh** [comment]
I implemented a fix and pushed it to both sites on the server.  This will require you losing and reestablishing your connection to the server to see the change.  I ran a couple of models, but did not test all possible combinations.  This fix does require that observations of harvest and tags are up to date to the same year.  I will leave this open and think about a more robust solution that might do partial prediction if one of the databases is lagging.
